check.ps1

```
@@ -201,186 +190,117 @@ function Get-JsonObject {
   }
 }
 
-function ConvertTo-VgoStringArray {
-  param($Value)
+function Normalize-ComparablePath {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'Normalize-ComparablePath' uses an unapproved verb.

(PSUseApprovedVerbs)